tests/cmds/test_users.py

```diff
@@ -1356,3 +1402,137 @@ def test_orgs_show_when_invalid_org_uid_raises_error(runner, cli_state, custom_e
     result = runner.invoke(cli, ["users", "orgs", "show", TEST_ORG_UID], obj=cli_state)
     assert result.exit_code == 1
     assert f"Invalid org UID {TEST_ORG_UID}." in result.output
+
+
+def test_add_cloud_alias_calls_add_cloud_alias_with_correct_parameters(
```

Comment:
In general we often add test cases for the commands if they return any specific py42 errors.  Looking at the py42 methods, looks like `add_cloud_alias` will throw an error if the name is too long, so we'd probably want an `add-alias` case that checks for that error message.

Reply:
Since these methods also implement the `sdk.get_user()` py42 method, we also want to check that we return a helpful error if that user doesn't exist or the customer doesn't have permission to view them

Reply:
I'm going to switch this method to use the `users` module of the sdk for consistency, and so we can use an existing CLI error we have for this use case.

Reply:
So it does look like the aliases are applicable to all users regardless of if theyre on a watchlist, so maybe this is more a py42 structure question, disregard my previous comments, lets leave these in the user group.

Reply:
Yeah the structure is definitely a little bit odd, since the methods to add aliases exist in the context of detection lists, but they are used in other parts of Incydr as well. I think it makes sense to have these in the `users` command in the CLI because of that broader use, even though the backend is dealing with the detection lists service.